Source/ThirdParty/ANGLE/src/libANGLE/renderer/gl/FramebufferGL.cpp

```diff
@@ -1084,8 +1084,8 @@ angle::Result FramebufferGL::adjustSrcDstRegion(const gl::Context *context,
 
         GLuint sourceXHalvings = 0;
         GLuint sourceYHalvings = 0;
-        GLint sourceOriginX    = bounds.sourceRegion.x;
-        GLint sourceOriginY    = bounds.sourceRegion.y;
+        GLint sourceOriginX; // Removed unneeded initializer to placate clang static analyzer.
```

Comment:
Not entirely sure what's the best approach here. Maybe move the declarations downwards near their initialition, so it's obvious it's initialized in all cases? The comment should be removed..